client/src/models/validations/validator.js

```diff
@@ -0,0 +1,9 @@
+export default class Validator {
```

Comment:
Are you using this class exclusively for form validation? If so, you could make it a bit more obvious with a better name. `FormValidator`?

Reply:
Yep that's all it's being used for. I should probably also rename the folder to something form related